src/VirtualClient/VirtualClient.Actions/Sysbench/sysbench scripts/cleanup-database.py

```diff
@@ -0,0 +1,22 @@
+import subprocess, argparse, os
```

Comment:
Can we put these scripts in VC build and not mysql package? This way you make sure the code and script match in the same version, you don't have situation people running rev2 in their profile with new VC